src/SmoFacade/Database.cs

```diff
@@ -50,6 +50,20 @@ public void DropAssociatedLogins()
         });
     }
 
+    public void AddUser(UserProperties user)
+    {
+      var matchingUser =
+        Users.SingleOrDefault(u => u.Name.Equals(user.Name, StringComparison.InvariantCultureIgnoreCase)) ??
+        new User(user, _server, this);
+    }
```

Comment:
You could return back the user - will allow the caller to run consecutive functions?

`db.AddUser("...").DoSomething()`